lib/setup.py

```diff
@@ -37,7 +37,7 @@
     "importlib-metadata>=1.4",
     "numpy",
     "packaging>=14.1",
-    "pandas<2,>=0.25",
+    "pandas>=0.25",
```

Comment:
We are kind of checking with our Python min tests for older Pandas versions since it's still running on Python 3.7 and Pandas 2.x will not be available for 3.7. But I think this is only a side-effect of us supporting a mostly outdated Python version. I think we should prioritize implementing a way for automated tests with older versions of our dependency. I believe we could get a simple approach implemented without too much effort.

> How confident are we that we won't do this

I'm pretty confident that I will not integrate new Pandas stuff, but I there is of course a risk (but already even with Pandas 0.25 vs 1.0) and not all changes are obvious in Pandas.